src/test/java/com/linkedin/kafka/clients/producer/DocumetationTest.java

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright 2017 LinkedIn Corp. Licensed under the BSD 2-Clause License (the "License").  See License in the project root for license information.
+ */
+package com.linkedin.kafka.clients.producer;
+
+import com.linkedin.kafka.clients.consumer.ExtensibleConsumerRecord;
+import com.linkedin.kafka.clients.utils.HeaderKeySpace;
+import com.linkedin.kafka.clients.consumer.LiKafkaConsumerImpl;
+import com.linkedin.kafka.clients.utils.tests.AbstractKafkaClientsIntegrationTestHarness;
+import java.util.Collections;
+import java.util.Properties;
+import java.util.concurrent.Future;
+import org.apache.kafka.clients.consumer.Consumer;
+import org.apache.kafka.clients.consumer.ConsumerConfig;
+import org.apache.kafka.clients.consumer.ConsumerRecord;
+import org.apache.kafka.clients.consumer.ConsumerRecords;
+import org.apache.kafka.clients.producer.Producer;
+import org.apache.kafka.clients.producer.ProducerConfig;
+import org.apache.kafka.clients.producer.RecordMetadata;
+import org.apache.kafka.common.TopicPartition;
+import org.apache.kafka.common.serialization.StringDeserializer;
+import org.apache.kafka.common.serialization.StringSerializer;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+
+/**
+ * This is a simple code example we use in the documentation.
+ */
+public class DocumetationTest extends AbstractKafkaClientsIntegrationTestHarness {
```

Comment:
So maybe DocumentationExampleTest? It is not testing the documentation itself, because there is actually documentation test in Kafka which tests the auto generated documentation for configurations.